source/develop/release-management/features/gluster-self-heal-monitoring.html.md

```diff
@@ -27,7 +27,7 @@ Gluster Self Heal helps to heal data on the gluster bricks when there are some i
 *   Status: In development.
 
 ## Monitoring Self Heal
-We will monitor self heal status at the volume level. ‘gluster volume heal VOLNAME info’ command will be used to get the heal info. If there are some unsynced entries in the volume then volume will be marked as ‘Needs Healing’ and bricks which are having unsynced entries will be marked accordingly. Ovrit will sync self heal info for all the gluster volumes with frequency of 10 minutes.
+We will monitor self heal status at the volume level. ‘gluster volume heal VOLNAME info’ command will be used to get the heal info. If there are some unsynced entries in the volume then volume will be marked as ‘Needs Healing’ and bricks which are having unsynced entries will be marked accordingly. Ovirt will sync self heal info for all the gluster volumes with frequency of 10 minutes. Sync frequency is configurable using VDC Option 'GlusterRefreshRateHealInfo'.
```

Comment:
I'd change following:

1. OVirt -> oVirt

2. Synchronization frequency can be changed using engine-config option 'GlusterRefreshRateHealInfo'.